### PR TITLE
F/5784 refactor release

### PR DESCRIPTION
**Pull request by sansth1010**
Prepare for release  [5784](https://devtopia.esri.com/dc/hub/issues/5784)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # CHANGELOG.md
 
+## 3.0.0
```
Comment (rgwozdz) [inline CHANGELOG.md:3]: This should say `Unreleased`.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@koopjs/koop-provider-hub-search",
-  "version": "2.1.1",
+  "version": "3.0.0",
```
Comment (rgwozdz) [inline package.json:3]: don't change the version number in this PR. We merge to master, then we will do a specific release commit on master where we update the package.json.

**Review by rgwozdz — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (sansth1010) [reply CHANGELOG.md]: Changed the version to `Unreleased`

File: package.json (hunk 2 above)
Comment (sansth1010) [reply package.json]: Got it! I have reverted the version bump. Will update version once merged.

**Review by rgwozdz — APPROVED** [review]

**Pull request merged**